src/elements/content-sidebar/activity-feed/task-new/__tests__/__snapshots__/Task-test.js.snap

```diff
@@ -114,27 +114,43 @@ exports[`elements/content-sidebar/ActivityFeed/task-new/Task should correctly re
     onSubmitSuccess={[Function]}
     taskFormProps={
       Object {
-        "approverSelectorContacts": Array [
+        "approverSelectorContacts": undefined,
+        "approvers": Array [
           Object {
-            "id": "1",
-            "item": Object {},
-            "name": "Jake Thomas",
-            "text": "Jake Thomas",
-            "type": "user",
-            "value": "1",
+            "id": "current-user-assignment-id",
+            "permissions": Object {
+              "can_delete": true,
+              "can_update": true,
+            },
+            "role": "ASSIGNEE",
+            "status": "NOT_STARTED",
+            "target": Object {
+              "id": "1",
+              "name": "Jake Thomas",
+              "type": "user",
+            },
+            "type": "task_collaborator",
           },
           Object {
-            "id": "3",
-            "item": Object {},
-            "name": "Patrick Paul",
-            "text": "Patrick Paul",
-            "type": "user",
-            "value": "3",
+            "id": "other-user-assignment-id",
+            "permissions": Object {
+              "can_delete": true,
+              "can_update": true,
+            },
+            "role": "ASSIGNEE",
+            "status": "COMPLETED",
+            "target": Object {
+              "id": "3",
+              "name": "Patrick Paul",
```

Comment:
We should probably avoid names of Boxers in our tests.

Reply:
Probably! there are definitely some instances of names in the test data. here, I changed the snapshot which displays a bit more info than before